hook-producer/src/main.rs

```diff
@@ -21,10 +21,24 @@ async fn listen(app: Router, bind: String) -> Result<()> {
 async fn main() {
     tracing_subscriber::fmt::init();
 
-    let app = handlers::router();
-
     let config = Config::init_from_env().expect("failed to load configuration from env");
 
+    let pg_queue = PgQueue::new(
+        // TODO: Coupling the queue name to the PgQueue object doesn't seem ideal from the producer
+        // side, but we don't need more than one queue for now.
+        &config.queue_name,
+        &config.table_name,
+        &config.database_url,
+        // TODO: It seems unnecessary that the producer side needs to know about the retry policy.
+        RetryPolicy::default(),
```

Comment:
Yeah, we can move this only to the consumer.

Reply:
Cool, it's not a big deal or anything, I'll leave the TODO for later.